velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -562,7 +562,11 @@ void ParquetRowReader::filterRowGroups() {
     auto fileOffset = rowGroups_[i].__isset.file_offset
         ? rowGroups_[i].file_offset
         : rowGroups_[i].columns[0].file_offset;
-    VELOX_CHECK_GT(fileOffset, 0);
+    VELOX_CHECK_GE(fileOffset, 0);
+    if (fileOffset == 0) {
```

Comment:
We still need to pass the metadata filters in this case, do not shortcut this one

Reply:
Thanks, just fixed.